documentation/docs/configuration/yaml_config.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Configuration YAML
+
+Each confiugraion yaml contains a list of configurations to be deployed.
+
+A basic configuraion yaml looks something like this
+
+```yaml
+configs:
+- id: test-dashboard
+  config:
+    name: Test Dashboard
+    template: dashboard.json
+    parameters:
+      owner: Test User
+```
+
+As you can see the top level element is `configs`. Its value is a list of
+configurations.
+
+Each configuration requres a number of fields. The first field is `id`,
+Then there is the config field. Those two fields are requried. it is also
+possible to override values from config on a per group and environment
+level. For this, there exists the `groupOverrides` and `environmentOverrides`
+fields.
+
+## ID
+
+The `id` field is used to identify a config within the configurations. It
+has to be unique for on an api level per project. So it is possible to have
+e.g. two dashboards with the same `id` in two different projects.
+
+It is important to note, that the field is only local to monaco. It has nothing
+to do with the id provided by the dynatrace api.
+
```

Comment:
add that the ID is used when referencing a config's parameters in another config?